test/testrelative.c

```diff
@@ -112,8 +156,12 @@ int main(int argc, char *argv[])
         SDL_RenderClear(renderer);
     }
 
-    if (SDL_SetRelativeMouseMode(SDL_TRUE) < 0) {
-        return 3;
+    if (!warp) {
```

Comment:
Can you invert this test, e.g. if (positive test) { do positive thing }

Reply:
... and add a comment explaining what we're planning to do here?

Reply:
Done.